src/flownet/config_parser/_config_parser.py

```diff
@@ -19,10 +25,40 @@ def create_schema(_to_abs_path) -> Dict:
         Dictionary to be used as configsuite type schema
 
     """
+
+    @configsuite.transformation_msg("Convert string to lower case")
+    def _to_lower(input_data: Union[List[str], str]) -> Union[List[str], str]:
+        if isinstance(input_data, str):
+            return input_data.lower()
+
+        return [x.lower() for x in input_data]
+
+    @configsuite.transformation_msg("Tries to convert input to absolute path")
```

Comment:
Always wondered about why this is so vaguely written. Why not "Convert path to absolute path" - or something like this.

It is not really trying - it's doing it otherwise it crashes....`?